components/linkButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {Linking, StyleSheet, Text, View} from 'react-native';
+import {Colors, GlobalStyles} from '../styles';
+
+export default function LinkButton({title, url}) {
+  return (
+    <View style={styles.content}>
```

Comment:
View/Text with the onPress handler on Text is not the ideal way to implement a custom button- Text does not render the appropriate visual feedback on press as expected. If you look at how the existing Button component is implemented, it uses TouchableOpacity so that you see the "pressed" feedback animated on the button upon interaction.

Which begs the question- why re-implement a button component and all its styling when we already have one done? You can just import and use the Button component that has already been implemented here.